etc/cas.properties

```diff
@@ -61,6 +61,12 @@ cas.callutheran.prefix.url=https://sso.callutheran.edu/cas/
 cas.callutheran.client.name=callutheran
 cas.callutheran.cas.protocol=SAML
 #
+# CAS Client: Concordia College
+cas.cord.login.url=https://bprdeis.cord.edu:8443/cas/login
+cas.cord.prefix.url=https://bprdeis.cord.edu:8443/cas/
+cas.cord.client.name=cord
+cas.cord.cas.protocol=SAML
```

Comment:
* Step 4: add settings for the CAS client bean configuration
  * Step 4+: add the settings to the charts, see https://github.com/CenterForOpenScience/helm-charts/pull/57